### ISSUE TITLE
[Payment Issue]: 

### ISSUE THREAD

**Issue by davir-a** [soonaverse/soonaverse-dao#921]
### Discord Name

@dia#7165

### Soonaverse Profile Id

0x94EBf7dc2a86f1e81eABC0023ffa6AbD669F1405

### Payment Action

Other (Please describe in comments)

### Payment Problem

Other (Please describe in comments)

### Wallet Type

Other (Please describe in comments)

### Wallet Version

firefly shimmer 2.0.1

### Payment Sent to Address

smr1qqr6vywatgplrr4a55l8gejs8hkuuvha47xuh77vjaema9rh5dqp7at8n5y

### Amount Sent

.0496smr

### Payment Message Id

0x96a37773f3360e8a3ea98917e8707241d65c3a07c879b613afaf738d9b6e0f38

### Comments

```shell
tried to claim $fee to verified smr address smr1qprf885t4890f2trzyh4dzcceh4cqavme6jqlnv9e66hy3m4hcg72qryjdy

claim was "successful". 

$fee disappeared from soonaverse profile.

does not show up in firefly shimmer wallet or in explorer

https://explorer.shimmer.network/shimmer/addr/smr1qprf885t4890f2trzyh4dzcceh4cqavme6jqlnv9e66hy3m4hcg72qryjdy
```

**Comment by adamunchained** [comment]
There was just a slight delay in processing it. It seems to be processed now.

https://explorer.shimmer.network/shimmer/block/0x418f1b54af1b3cc733a48e217a8ea0a8715263000ff7b7ca8ddc0d8db6f20459

**Comment by davir-a** [comment]
maybe its me but i'm still not seeing any $fee in firefly shimmer nor do i see the .0496smr transaction or the $fee on explorer.
https://explorer.shimmer.network/shimmer/addr/smr1qprf885t4890f2trzyh4dzcceh4cqavme6jqlnv9e66hy3m4hcg72qryjdy

appreciate the fast response and by no means is this an urgent matter, more curiosity.

thanks

**Comment by adamunchained** [comment]
You're right, I just noticed the transaction is actually conflicting. We're looking into this and I'll let you know by tomorrow. 

It seems the issue is due a transaction with "storage deposit lock" and Soonaverse is properly not unlocking it.

**Comment by adamunchained** [comment]
@boldizsar-unchained - fyi this is one of the transaction that I've mentioned in slack.

**Comment by adamunchained** [comment]
@davir-a - this has been now fixed, you should see your FEE in the wallet.
